doc/dev/crimson/crimson.rst

```diff
@@ -175,6 +227,11 @@ The following options can be used with ``vstart.sh``.
 ``--bluestore``
     Use the alienized BlueStore as the object store backend. This is the default.
 
+``--crimson-smp``
+    BlueStore only.  The number of cores to use for each OSD.
```

Comment:
I have updated this text, but I suspect that I am leaving something out here that I don't quite understand. I'll make sure that @cyx1231st and @athanatos have been consulted and satisfied before this is merged.

Reply:
Looks good.

> The balance of available cores (as determined by `nproc`) will be assigned to BlueStore.

I suppose this might be changed after @Matan-B 's change.